tests/drivers/led/led_api/src/main.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020 Seagate Technology LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <ztest.h>
+
+#include "test_led_api.h"
+
+void test_main(void)
+{
+	ztest_test_suite(led_test,
```

Comment:
we don't generally implement separate suites for user vs non-user, just do the user variant. user mode code path is a superset of the supervisor code path, and devices that don't support user mode just run everything in supervisor mode.

in other words, delete this first suite, the second is suitable for everyone.

Reply:
OK I'll do. I got it wrong from another driver test.